src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/ChangeLog.md

```diff
@@ -19,6 +19,48 @@
 -->
 ## Current Release
 
+## Version 0.4.0
+* New cmdlet
+	* Get-AzsDrive
+	* Get-AzsVolume
+	* Get-AzsStorageSubSystem
+* Unsupported cmdlet
```

Comment:
What does "Unsupported cmdlet" mean?

Reply:
We changed the resource path, properties and PS cmdlet name for volume, the Get-AzsVolume cmdlet can get all info that Get-AzsInfrastructureVolume can provide before, so we decided to remove Get-AzsInfrastructureVolume cmdlet.

Reply:
You should change this to say "Removed cmdlet" then, or say "This cmdlet has been removed, please use Get-AzsVolume".

Reply:
Fixed.